### ISSUE TITLE
com.docker.network.bridge.host_binding_ipv6 without effect

### ISSUE THREAD

**Issue by heinemml** [robbertkl/docker-ipv6nat#1]
I'm trying to setup ipv6nat on my machine.

I need a dedicated IPv6 address for the NAT as I want my main machine to be reachable on port 22 at its own IP.

If I'm reading the documentation correctly I can only define this on a user-defined network. Using the default bridge would probably easier but I don't see how i could set `com.docker.network.bridge.host_binding_ipv6` there.

Now I created  a user-defined network as documented in Option B. But when I do a `ip6tables -L -t nat -v` I see the following result:
``` 
Chain DOCKER (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 RETURN     all      br-d152d505a932 any     anywhere             anywhere            
    0     0 DNAT       tcp      !br-d152d505a932 any     anywhere             anywhere             tcp dpt:ssh to:[fd00:dead:babe::2]:22
```

If I compare it to the ipv4 entry I see in the destination that there should be the defined host binding IP instead of anywhere. 

If I inspect the created network I see that "com.docker.network.bridge.host_binding_ipv6" is set to the correct IP.

Any idea what could be wrong?

**Comment by heinemml** [comment]
I think I figured it out.

In the Readme the option is called  `com.docker.network.bridge.host_binding_ipv6` which matches the docker setting `com.docker.network.bridge.host_binding_ipv4`
But if I look in the code here https://github.com/robbertkl/docker-ipv6nat/blob/master/state.go#L172 I see that the option is called  `com.docker.network.bridge.binding_ipv6`.

The second error I had was adding a subnet to the IP (like `/80`). I was mislead by the mentioning of `::/0` in the docs.

So it seems to work now, but the documentation or the code needs fixing.

**Comment by robbertkl** [comment]
Hi, you are exactly right, on both accounts. The name should match the IPv4 version, I'll make the necessary fixes to code and/or documentation right away, including your comment about the subnet.

I'm actually not using the option myself, for exactly the reason you mentioned before: the default `bridge` network is created on docker startup and it's options come from: `--ip=x.x.x.x` translates into `com.docker.network.bridge.host_binding_ipv4=x.x.x.x` on the `bridge` network). Unfortunately, there is no way to do this for my "user-defined" IPv6 bind option.

My server actually has just 1 IPv6 address configured, so the default bind to `::` is fine for me.

**Comment by robbertkl** [comment]
All fixed in v0.1.2; new image is on Docker Hub as well.

Thanks for reporting @heinemml !
